interfaces/mount/change.go

```diff
@@ -42,6 +44,29 @@ type Change struct {
 	Action Action
 }
 
+// Perform executes the desired mount or unmount change.
```

Comment:
Ok, well, this still suggests to me that this code should live somewhere else (perhaps even just mount_utils.go file in this directory) or somewhere since you are mixing library code meant to be used only by update-ns with code meant to be used by the interfaces system.

I've said what I have to say on this; if you feel strongly the code is in the correct place, then perhaps at the very least group the update-ns library code functions together with a comment that these are helpers for update-ns.

If other reviewers are fine with this PR, I won't block on it any more.